### DISCUSSION TITLE
Warnings in chrome devtools console when using Menu component

### DISCUSSION THREAD

**Discussion by egorshubin** [primefaces/community#761] [PrimeVue]
I use component Menu with popup: https://primevue.org/menu/#popup 
It works, but gives a lot of warnings in console: 
[Vue warn]: Failed to resolve component: router-link
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Menuitem key="Rename current preset2" id="overlay_menu_2" item= {label: 'Rename current preset', icon: 'pi pi-pencil', command: ƒ}  ... > 
  at <BaseTransition onEnter=fn onLeave=fn<onLeave> onAfterLeave=fn<bound onAfterLeave>  ... > 
  at <Transition name="p-connected-overlay" onEnter=fn<bound onEnter> onLeave=fn<bound onLeave>  ... > 
  at <Portal appendTo="body" disabled=false > 
  at <Menu ref="menu" id="overlay_menu" model= (3) [{…}, {…}, {…}]  ... > 
  at <ChoosePreset> 
  at <PanelOne> 
  at <App>

**Comment by tugcekucukoglu** [comment]
Menu and other menu components do not support using vue-router as a dependency. We suggest to investigate [Router example](https://primevue.org/menu/#router)

**Reply by egorshubin** [reply]
Thanks, but I just didn't do anything else than was written in documentation. Why doesn't it work properly out-of-the-box? Without manual templating
